joomla/joomla-cms
# [installation] Created in 2011-01-01? Update created date plus minor corrections

#### Summary of Changes

Joomla on install articles, categories etc with a created date with more than 5 years (2011-01-01) which does not look good when you are installing something new.

So this PR adds a method to the installation process to update core and sample data created dates to the current date.

Also corrects category creator id in com_banners and com_user_notes (was missing).
#### Testing Instructions
1. Use latest staging
2. Delete configuration.php, refresh the page and install joomla with any sample data install step
3. Check the created dates of articles, categories, contacts, etc
4. Now apply this patch, delete configuration.php, refresh the page and install joomla
5. Repeat step 2 to 3 check the created date is now the Joomla install date.
6. Do a code review.
